checkstyle.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+ 
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+    <module name="NewlineAtEndOfFile" />
+    <module name="TreeWalker">
+        <module name="ImportOrder">
+            <property name="option" value="bottom" />
+        </module>
+    </module>
+</module>
```

Comment:
Newline at end of file 😭

Reply:
this is out of the context to apply the check style otherwise also resources should be involved (and this could not be always possible). Instead it is true for java files.